package.json

```diff
@@ -33,8 +33,10 @@
     "start:playground": "FUNDAMENTAL_REACT_PLAYGROUND=true npm start",
     "std-version": "standard-version -m \"chore(release): version %s build ${TRAVIS_BUILD_NUMBER} [ci skip]\"",
     "test:coverage": "jest --coverage",
+    "test:coverage:watch": "jest --coverage --watch",
     "test:coveralls": "jest --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
     "test:dev": "jest",
+    "test:watch": "jest --watch",
     "test": "node scripts/test.js",
```

Comment:
this was discussed in in #381. 

`test` is running in a watch mode. So what does `node scripts/test.js` give us that `jest --watch` doen't?

if the answer is nothing, we *could* delete that script and do 

```
test:dev: ''jest"
test:watch: "jest --watch"
test: "npm run test:watch"
```

Reply:
The output of the two commands (`npm test` using node scripts/test.js or `npm run test:watch`) look identical and based off the conversation from #381 the change to `npm run test:watch` returns the watch functionality, I updated the package.json file as well as delete the scripts/test.js file